tools/base/BUILD

```diff
@@ -0,0 +1,46 @@
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+load("//bazel:envoy_build_system.bzl", "envoy_package")
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+py_library(
+    name = "checker",
+    srcs = ["checker.py"],
+)
+
+py_library(
+    name = "utils",
+    srcs = ["utils.py"],
+)
+
+py_library(
+    name = "test_utils",
+    srcs = ["tests/test_utils.py"],
+)
+
+# do this better
+py_binary(
+    name = "pytest_checker",
+    srcs = [
+        "pytest_checker.py",
```

Comment:
so for now this adds a python file for each test target (which is kinda redundant)